packages/v/valigators.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "valigators",
+  "description": "A simple validation library for any data type",
+  "filename": "valigators.min.js",
+  "keywords": ["validation", "validators", "validate", "check", "alligator"],
+  "authors": [
+    {
+      "name": "Will Kelly"
+    }
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wkelly1/Valigators.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "valigators",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["**/*"]
```

Comment:
Please avoid using open globstar patterns, as they are dangerous and can lead to cdnjs cloning excess files that we don't need.

You likely just need something like `*.js` here.

Reply:
Ok thanks, I've updated `"files"` to just the path to the bundle file.